src/pages/TeamView/TeamView.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { useSelector } from 'react-redux';
+import { Box, Button, Heading, Input, FormControl} from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: Int!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const TeamView = () => {
+  const {
+    user: { teamId },
+  } = useSelector(state => state.auth);
+
+  return teamId ? (
+    <TeamOverview teamId={teamId} />
+  ) : (
+    <div>
+      <h1>Welcome</h1>
+      {'User is not part of a team (todo: implement this view)'}
+    </div>
+  );
+};
+
+const InviteTeamMembers = () => {
+  const [inputList, setInputList] = useState([{email: ""}]);
+
+  const handleInputChange = (e, index) => {
+    const { name, value} = e.target;
+    const list = [...inputList];
+    list[index][name] = value;
+    setInputList(list);
+  }
+
+  const handleRemoveClick = index => {
+    const list = [...inputList];
+    list.splice(index, 1);
+    setInputList(list);
+  }
+
+  const handleAddClick = () => {
+    setInputList([...inputList, {email: ""}])
+  }
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    const list = [...inputList];
+    let output = "";
+    for (let i = 0; i < list.length; i++) {
+      output = output.concat(list[i].email);
+    }
+    alert(output);
+  }
+
+  return (
+    <Box w="50%">
+      <form onSubmit={e => handleSubmit(e)}>
+        {inputList.map((x, i) => {
+          return (
+            <Box d="flex" mb="8px">
```

Comment:
anytime a `map` is used, we should use `key` attribute (if you peep the dev tools you should see a React warning if `key` attribute is not used)